Gemfile

```diff
@@ -16,6 +16,7 @@ gem 'sidekiq'
 # Security
 gem 'bcrypt'
 gem 'jwt'
+gem 'googleauth', require: false
```

Comment:
Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `googleauth` should appear before `jwt`.